installation/configuration.md

```diff
@@ -120,77 +141,32 @@ rewrite ^/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2;
 rewrite ^/([^/\.]+)/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2&id=$3;
 ```
 
-This and by desabling the 
+## Theme Configuration
+
+Leantime allows you to white label the entire system by changing the color scheme of your site, set your own logo, and also set primary and secondary color. This can be easily done by updating the `config/.env` file:
 
-## Language
-To change the default language of your installation just update   
 ```php
-public $language = 'en-US'; 
+LEAN_LOGO_PATH='/dist/images/logo.svg'           # Default logo path, can be changed later
+LEAN_PRINT_LOGO_URL='/dist/images/logo.jpg'      # Default logo URL use for printing (must be jpg or png format)
+LEAN_DEFAULT_THEME='default'                     # Default theme
+LEAN_PRIMARY_COLOR='#1b75bb'                     # Primary Theme color
+LEAN_SECONDARY_COLOR='#81B1A8'                   # Secondary Theme Color
 ```
-to a language string available. You can see all available languages in `resources\languages\languagelist.ini`
 
-## Timezone
-To change the default timezone of your installation, just update
+These settings can also be changed within Leantime under Account->Theme.
+
+## Sitename, Language and Timezone
+
+Configure sitename, your default language and timezone by updating the default settings section in `config/.env`:
+
 ```php
-public $defaultTimezone = 'America/Los_Angeles';
+LEAN_SITENAME='Leantime'                         # Name of your site, can be changed later
+LEAN_LANGUAGE='en-US'                            # Default language
+LEAN_DEFAULT_TIMEZONE='America/Los_Angeles'      # Set default timezone
 ```
-to a valid timezone . You can look up all the timezones at https://www.php.net/manual/en/timezones.php
 
-## ENV Vars
-The entire configuration can also be passed in as env vars (at docker startup or server variables). The list of equvalent variables is:
-```
-LEAN_SITENAME
-LEAN_LANGUAGE
-LEAN_MAIN_COLOR
-LEAN_LOGO_PATH
-LEAN_APP_URL
-LEAN_DEFAULT_TIMEZONE
-
-/* Database */
-LEAN_DB_HOST
-LEAN_DB_USER
-LEAN_DB_PASSWORD
-LEAN_DB_DATABASE
-
-/* Fileupload */
-LEAN_USER_FILE_PATH
-
-LEAN_USE_S3
-LEAN_S3_END_POINT
-LEAN_S3_KEY
-LEAN_S3_SECRET
-LEAN_S3_BUCKET
-LEAN_S3_USE_PATH_STYLE_ENDPOINT
-LEAN_S3_REGION
-LEAN_S3_FOLDER_NAME
-
-/* Sessions */
-LEAN_SESSION_PASSWORD
-LEAN_SESSION_EXPIRATION
-
-/* Email */
-LEAN_EMAIL_RETURN
-LEAN_EMAIL_USE_SMTP
-LEAN_EMAIL_SMTP_HOSTS
-LEAN_EMAIL_SMTP_USERNAME
-LEAN_EMAIL_SMTP_PASSWORD
-LEAN_EMAIL_SMTP_AUTO_TLS
-LEAN_EMAIL_SMTP_SECURE
-LEAN_EMAIL_SMTP_PORT
-
-/*ldap*/
-LEAN_LDAP_USE_LDAP
-LEAN_LDAP_LDAP_TYPE
-LEAN_LDAP_HOST
-LEAN_LDAP_PORT
-LEAN_LDAP_BASE_DN
-LEAN_LDAP_DN
-LEAN_LDAP_USER_DOMAIN
-LEAN_LDAP_KEYS
-LEAN_LDAP_GROUP_ASSIGNMENT
-LEAN_LDAP_DEFAULT_ROLE_KEY
-```
 ## OpenID-Conenct (OIDC) Configuration
+
 Warning! This is still experimental and may not be a 100% secure.
```

Comment:
@marcelfolaron what's missing here that we're saying it's not 100% secure? I can't remember if OIDC was maintained by us?

Reply:
hmm. yeah @ctanveer please take this out